src/libcore/num/mod.rs

```diff
@@ -1601,6 +1601,29 @@ $EndFeature, "
             }
         }
 
+        doc_comment! {
+            concat!("Wrapping (modular) absolute value. Computes `self.abs()`, without any wrapping
+or panicking.
+
+
+# Examples
+
+Basic usage:
+
+```
+", $Feature, "#![feature(unsigned_abs)]
+assert_eq!(100", stringify!($SelfT), ".unsigned_abs(), 100", stringify!($UnsignedT), ");
+assert_eq!((-100", stringify!($SelfT), ").unsigned_abs(), 100", stringify!($UnsignedT), ");
+assert_eq!((-128i8).unsigned_abs(), 128u8);",
+$EndFeature, "
+```"),
+            #[unstable(feature = "unsigned_abs", issue = "none")]
```

Comment:
You'll need to create a tracking issue.

Reply:
Do I do this now? Or do I have to wait for the PR to get merged (some of the ones I looked at recently waited)?

Reply:
I'd do it now, that way you don't need to submit two PRs.